### ISSUE TITLE
Melhorar a responsividade 

### ISSUE THREAD

**Issue by fabriciorissetto** [bernardobrezende/gh-x9#20]
A listagem de commits estrapola o tamanho da tela em dispositivos pequenos. A sugestão simples é ocultar a coluna dos comentários do commit em resoluções pequenas.

**Comment by bernardobrezende** [comment]
Concordo. Mas só pra esclarecer: ocultar a coluna de comentários ou commits?

Atualmente ainda não temos a coluna de comentários.

**Comment by fabriciorissetto** [comment]
A de commit.

**Comment by bernardobrezende** [comment]
Ah sim. Ou esconder a "última atualização", pq a do commit é realmente a mais importante, não? Digo, geralmente eu entro no sistema para clicar em algum link nela contido.

Uma melhoria que eu acabei fazendo no PR que está ativo é juntando as duas primeiras colunas em uma: Imagem e usuário do candidato.
